chipy_org/apps/job_board/models.py

```diff
@@ -1,3 +1,55 @@
 from django.db import models
+from django.utils import timezone
+from chipy_org.libs.models import CommonModel
+from chipy_org.apps.sponsors.models import Sponsor
 
-# Create your models here.
+MAX_LENGTH = 255
+STATUS_CHOICES = [('pending', 'Pending Approval'),
+                   ('approved', 'Approved'),
+                   ('denied', 'Denied'),
+                   ('extended', 'Extended'),
+                   ('archived', 'Archived') 
+                 ]
+
+
+class JobPost(CommonModel):
+    
+    __original_status = None
+
+    position = models.CharField(max_length=MAX_LENGTH)
+    company_name = models.CharField(max_length=MAX_LENGTH)
+    description = models.CharField(max_length=2500)
+
+    is_sponsor = models.BooleanField(
+        default=False, verbose_name="Is the company a sponsor of ChiPy?")
+
+    company_sponsor = models.ForeignKey(Sponsor, blank=True, null=True)
+
```

Comment:
Can this just be called sponsor?   Doing so would make our templates more consistent and intuitive.

Reply:
Hey @elmq0022,  Aly suggested that I take out the sponsor dropdown menu so the 'company_sponsor' field will be removed.  You can read his comment below. Also, in a comment below to Aly, I explain how 'status_change_date' works, which might be helpful for you to know how that works.